modules/op-viewing-pipeline-logs-in-kibana.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+// cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc
+//
+
+[id="op-viewing-pipeline-logs-in-kibana_{context}"]
+= Viewing pipeline logs in Kibana
+
+To view pipeline logs in the Kibana web console:
+
+.Procedure
+
+. Log in to {product-title} web console as a cluster administrator.
+.. In the top right of the menu bar, click the *grid* icon → *Observability* → *Logging*. The Kibana web console is displayed.
```

Comment:
```suggestion
. In the top right of the menu bar, click the *grid* icon → *Observability* → *Logging*. The Kibana web console is displayed.
```
I think this should just be a next step instead of a sub-task